README.md

```diff
@@ -1140,6 +1122,74 @@ ANSWERS.addComponent('FilterOptions', {
 });
 ```
 
+The options config varies depending on whether the optionType is 'STATIC_FILTER' or 'RADIUS_FILTER'.
+
+##### STATIC_FILTER
+
+```js
+{
+  options: [
+    {
+      // Label to show next to the filter option
```

Comment:
Can these comments have a "Required" or "Optional" prefix?

Reply:
updated! Reordered things to have required options first